packages/mds-stream-processor/labelers/geography-labeler.ts

```diff
@@ -15,29 +15,59 @@
  */
 
 import db from '@mds-core/mds-db'
-import { UUID, Nullable, Telemetry } from '@mds-core/mds-types'
+import { UUID, Nullable, Telemetry, Geography } from '@mds-core/mds-types'
 import { pointInShape } from '@mds-core/mds-utils'
+import getBbox from '@turf/bbox'
+import { BBox } from '@turf/helpers'
 import { MessageLabeler } from './types'
 
 export interface GeographyLabel {
   geography_ids: UUID[]
 }
 
-export const GeographyLabeler: () => MessageLabeler<
-  { telemetry?: Nullable<Telemetry> },
-  GeographyLabel
-> = () => async ({ telemetry }) => {
-  const gps = telemetry?.gps
-  if (gps) {
-    const { lat, lng } = gps
+type WithBbox<T> = T & { bbox: BBox }
 
-    const geographies = await db.readGeographies({ get_published: true })
+export const pointInBbox = ({ lat, lng }: { lat: number; lng: number }, bbox: BBox) => {
+  const [bottomLeft, topRight] = [
+    [bbox[0], bbox[1]],
+    [bbox[2], bbox[3]]
+  ]
```

Comment:
```ts
const [bottomLeftLng, bottomLeftLat, topRightLng, topRightLat] = bbox
```
Would be easier to read and also save you from having to use indexes on line `36`